consumer-server/src/logic/run-conversion.ts

```diff
@@ -151,7 +152,12 @@ export async function runConversion(
 
   // Run manifest builder before conversion if needed
   if (options.entityType === 'scene' && options.unityBuildTarget !== 'WebGL') {
```

Comment:
AB WebGL won't call this manifest generator, or it will fail on each call?

Reply:
We dont have the code to support ISS in webgl, so webGL asset bundles should build regularly.

Since there is no manifest, unity will do the non ISS flow